### ISSUE TITLE
Accessibility: allow users to skip filter list on product listing pages

### ISSUE THREAD

**Issue by philippsauer** [intershop/intershop-pwa#1824]
**Actual Behavior**

When browsing on product listing pages, users are required to tab through all filters before entering the relevant content on the listing page.

**Expected Behavior**

Users should be enabled to skip to relevant content when using the search or browsing through categories (Similar to the 'skip to main content'-link).

**Steps to Reproduce the Bug**

1. Go to https://intershoppwa.azurewebsites.net/
2. Search for "mouse"
3. Press tab key ~30 times to reach search results

[AB#107369](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/107369)
